pkg/httpserver/config.go

```diff
@@ -98,3 +100,8 @@ func file(path string) (string, error) {
 	}
 	return string(b), nil
 }
+
+func minify(body string) (string, error) {
+	b, err := json.Marshal(json.RawMessage(body))
```

Comment:
I think we should turn off EscapeHTML as to not have this modifying content outside of ordering and spacing.

Reply:
Pity jsonv2 is not here yet.